backend/handler/AuthHandler.ts

```diff
@@ -81,7 +81,7 @@ export function getTokenViaSamlResponse(samlResponse: string) {
 
 export function getAndCacheConfig(token?: string) {
     return jsonRequest<ConfigResponse>('GET', '/config', {
-        'Authentication-Token': token
+        'Authentication-Token': token as string
```

Comment:
Method `getAndCacheConfig` is being used in 3 places:
- backend/routes/Auth.ts
- backend/routes/ServerProxy.ts
- backend/handler/AuthHandler.ts

In all of those places, the token is being passed as a string
Because of that, I think that we should be able to modify it's parameter to always require a string
It'd slightly simplify this piece of code 😁 

 ```suggestion
 export function getAndCacheConfig(token: string) {
    return jsonRequest<ConfigResponse>('GET', '/config', {
        'Authentication-Token': token
 ```

Reply:
Yes. Looks like this function doesn't event make much sense with nullish token.
Applied.